.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -12,9 +12,8 @@ assignees: ''
 As clearly as you can, please tell us what the bug is.  
 
 Please note that Node.js is not supported by Connect-Web. We are working on
-Connect for Node.js - if you are interested, let us know on Slack or on 
+Connect for Node.js - if you are interested, let us know on [Slack](https://buf.build/links/slack) or on 
```

Comment:
Is a markdown link the right call here? 

This is what you see if you use the issue template: You are in the editor, and links can't be clicked. 

Maybe we should just remove the link altogether? 

<img width="714" alt="Screen Shot 2023-01-19 at 19 43 10" src="https://user-images.githubusercontent.com/4289451/213532409-5ddcda0c-4721-4468-9ad7-cc62872838c6.png">